blockchain/store_test.go

```diff
@@ -27,7 +28,7 @@ func makeTestCommit(height int64, timestamp time.Time) *types.Commit {
 	return types.NewCommit(types.BlockID{}, commitSigs)
 }
 
-func makeStateAndBlockStore(logger log.Logger) (sm.State, *BlockStore) {
+func makeStateAndBlockStore(logger log.Logger) (sm.State, *BlockStore, func()) {
```

Comment:
can we create an alias type for cleanup func?

```
// A cleanupFunc cleans up any config / test files created for a particular test.
type cleanupFunc func()
```

Reply:
done